packages/hint-doctype/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "ava": {
+    "failFast": false,
+    "files": [
+      "dist/tests/**/*.js"
+    ],
+    "timeout": "1m"
+  },
+  "description": "this hint checks if the HTML is using the most modern DOCTYPE.",
+  "devDependencies": {
+    "@hint/utils-tests-helpers": "^1.0.0",
+    "@types/debug": "0.0.31",
+    "@types/node": "10.9.1",
```

Comment:
This and some other ones need to be updated.

(I will take care of this once I merge)